api/apps/api_app.py

```diff
@@ -252,6 +254,8 @@ def get(conversation_id):
 
         conv = conv.to_dict()
         for referenct_i in conv['reference']:
+            if referenct_i is None or len(referenct_i) == 0:
+                continue
             for chunk_i in referenct_i['chunks']:

```

Comment:
for chunk_i in referenct_i.get('chunks', [])

Reply:
If the reference is a list type, it will still get an error.
'''
    for chunk_i in referenct_i.get('chunks', []):
                   ^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'get'
'''

Reply:
sry,you are right